urdf2webots/writeProto.py

```diff
@@ -473,8 +503,8 @@ def URDFShape(proto, link, level, normal=False):
             if name is None:
                 if visualNode.geometry.name is not None:
                     name = computeDefName(visualNode.geometry.name)
-            if visualNode.geometry.defName is None:
-                name = robotNameMain + '_' + name if robotNameMain else name
+            name = robotNameMain + '_' + name if robotNameMain else name
```

Comment:
We have to define the name before the condition, otherwise the adjusted path is not used, causing errors